server/odc-service/src/main/java/com/oceanbase/odc/service/dlm/utils/DataArchiveConditionUtil.java

```diff
@@ -58,6 +59,9 @@ private static String replaceVariables(String condition, Map<String, String> var
     }
 
     private static Map<String, String> getVariablesMap(List<OffsetConfig> variables) {
+        if (variables == null || variables.isEmpty()) {
```

Comment:
may simplify to `CollectionUtils.isEmpty(variables)`

Reply:
done